### DISCUSSION TITLE
Rainbow wallet extension

### DISCUSSION THREAD

**Discussion by naz3eh** [rainbow-me/rainbowkit#358] [General]
I've used the rainbow-kit lib and it is simply amazing. I wanted to know when can I expect the rainbow wallet extension

**Comment by nickbytes** [comment]
Thanks for the kind words @Nazeeh21 ! We're hoping to launch the extension sometime this year. 

Follow [@rainbowdotme](https://twitter.com/rainbowdotme) or [Mike](https://twitter.com/mikedemarais) to keep up to date on our progress!

**Comment by DanielSinclair** [comment]
The Rainbow extension is now in Beta!
